tests/console/mdadm.pm

```diff
@@ -41,6 +41,11 @@ sub run {
     upload_logs 'mdadm.log';
 }
 
+sub test_flags {
+# on SLE16 mdadm is older than SLE15 and thus is failing. Marking as non-fatal until it is resolved (last update 12-Feb-2025, https://bugzilla.suse.com/show_bug.cgi?id=1237075)
+    return {fatal => is_sle('>=16') ? 0 : 1};
```

Comment:
This marks the test as FATAL - meaning: all test modules run after this one won't run whenever mdadm fails.

Considering this test is already flaky (See https://progress.opensuse.org/issues/174751) this seems a very odd choice

Reply:
1. No. The test you linked has `QEMU_DISABLE_SNAPSHOTS=1` so even with fatal=0 it will abort the job.

Reply:
but this one also has that option enabled (https://openqa.suse.de/tests/16782810) but mdadm and quota are not blocking the others tests

Reply:
The logs of previous tests say:

```
[2025-02-12T08:06:04.773341Z] [debug] [pid:53262] stopping overall test execution because snapshotting is disabled/unavailable and "fatal => 0" has NOT been set explicitly
```

so explicitly fatal=0 overrides the snapshot_disabling causing the halt.

Reply:
> so explicitly fatal=0 overrides the snapshot_disabling causing the halt.

TIL.

Reply:
ah, me too then! Thanks Fabian